Dockerfile

```diff
@@ -1,19 +1,34 @@
-FROM ubuntu
+FROM ubuntu:18.04
 
-# System requirements
-RUN apt-get update && apt-get install -y \
-  build-essential \
-  curl \
-  python3-pip \
-  swig \
-  && rm -rf /var/lib/apt/lists/*
+WORKDIR /auto-sklearn
+
+# Copy the checkout autosklearn version for installation
+ADD . /auto-sklearn/
+
+# install linux packages
+RUN apt-get update
+
+# Set the locale
+RUN apt-get -y install locales
+RUN touch /usr/share/locale/locale.alias
+RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && locale-gen
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN apt install -y python3-dev python3-pip
+RUN pip3 install --upgrade setuptools
+RUN apt-get install -y build-essential curl 
+
+# https://github.com/automl/auto-sklearn/issues/314
+RUN apt-get install -y swig3.0
+RUN ln -s /usr/bin/swig3.0 /usr/bin/swig
 
 # Upgrade pip then install dependencies
 RUN pip3 install --upgrade pip
-RUN curl https://raw.githubusercontent.com/automl/auto-sklearn/master/requirements.txt \
-  | xargs -n 1 -L 1 pip3 install
+RUN pip3 install pytest==4.6.* pep8 codecov pytest-cov flake8 flaky openml
+RUN curl https://raw.githubusercontent.com/automl/auto-sklearn/master/requirements.txt | xargs -n 1 -L 1 pip3 install
+RUN pip3 install jupyter
 
 # Install
-RUN pip3 install \
-  auto-sklearn \
-  jupyter
+RUN pip3 install -e /auto-sklearn/
```

Comment:
Could you briefly explain what the `/` is doing here?

Reply:
Oh, so I wanted to make sure we use the latest image available.

Before in the docker file:

1. We did a change in the development repo. We install autosklearn from pip. You don't get the change in the development image.

Workflow now:
1- You checkout the development branch (or any branch in question) in /auto-sklearn of the container
2- You install autosklearn from the checkout version and successfully get the change you implemented.

Reply:
Got it, thanks.